### PR TITLE
Added sles15 resolved issue #320

### PR DESCRIPTION
**Pull request by FcoCalero**
Resolved Deprecated option ServerKeyBits for sles15
Added #320

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -171,17 +171,27 @@
...
+            case $::operatingsystemrelease {
+              /12\./: {
+                $default_sshd_config_subsystem_sftp = '/usr/lib/ssh/sftp-server'
+                $default_sshd_config_serverkeybits  = '1024'
+              }
+              /15\./: {
+                $default_sshd_config_subsystem_sftp = '/usr/lib/ssh/sftp-server'
+                $default_sshd_config_serverkeybits  = undef
+              }
+              default: {
```
Comment (ghoneycutt) [inline manifests/init.pp:190]: The default is the same as `12` so suggest using `default` and `15`.

**Comment by ghoneycutt** [comment]
Hi @FcoCalero Thanks for the contribution. This will also need tests added as well as rebased against master to pick up fixes for TravisCI.

Check out https://github.com/ghoneycutt/puppet-module-ssh/blob/master/spec/classes/init_spec.rb#L196

**Comment by FcoCalero** [comment]
Yeah sorry, i clicked it by accident and i didn't see the way to cancel it,
i will do something probably by next week or so.

File: manifests/init.pp [hunk 2]
```diff
@@ -176,18 +176,24 @@
...
       $default_sshd_config_include             = undef
       case $::architecture {
         'x86_64': {
-          if ($::operatingsystem == 'SLES') and ($::operatingsystemrelease =~ /^12\./) {
-            $default_sshd_config_subsystem_sftp = '/usr/lib/ssh/sftp-server'
-          } else {
-            $default_sshd_config_subsystem_sftp = '/usr/lib64/ssh/sftp-server'
+          if ($::operatingsystem == 'SLES') {
+            case $::operatingsystemrelease {
+              /15\./: {
```
Comment (anders-larsson) [inline manifests/init.pp:188]: Is it intentional to break SFTP support for SLE[DS] 12? It uses the wrong path after this change. Currently SLES12 is still stated as supported in metadata.json.

Sorry for writing in an old pull request though.

**Pull request merged**